lib/package.gd

```diff
@@ -691,6 +691,81 @@ DeclareGlobalFunction( "LoadPackageDocumentation" );
 ##
 ##  <Description>
 ##  loads the &GAP; package with name <A>name</A>.
+## <P/>
+##  As an example, the following loads the &GAP; package
+## <Package>Sonata</Package> (case insensitive) which provides methods for the construction and analysis of finite nearrings:
+##  <P/>
+##  <Log><![CDATA[
+##  gap> LoadPackage( "sonata" );
+##  #I  You may wish to install the xgap package
+##  #I  and enjoy the graphic capabilities of SONATA.
+##
+##   ___________________________________________________________________________
+##  /        ___
+## ||       /   \                 /\    Version 2.6
+## ||      ||   ||  |\    |      /  \               /\       Erhard Aichinger
+##  \___   ||   ||  |\\   |     /____\_____________/__\      Franz Binder
+##      \  ||   ||  | \\  |    /      \     ||    /    \     Juergen Ecker
+##      ||  \___/   |  \\ |   /        \    ||   /      \    Peter Mayr
+##      ||          |   \\|  /          \   ||               Christof Noebauer
+##  \___/           |    \|                 ||
+##
+##  System    Of   Nearrings     And      Their Applications
+##  Info: http://www.algebra.uni-linz.ac.at/Sonata/
+##
+##  true
+##  ]]></Log>
+##  <P/>
+##  The package name may be appropriately abbreviated as shown in the following
+##  two examples with the &GAP; packages <Package>Semigroups</Package>, for
+##  working with semigroups, monoids and associated structures, and
+##  <Package>Design</Package>, for working with block designs:
+##  <P/>
+##  <Log><![CDATA[
```

Comment:
This long example seems to be too verbose to me... Also, Semigroups is in active development, and eventually we would have to update the number. I suggest either to write in a plain text that `LoadPackage("semi")` loads Semigroups and `LoadPackage("d")` loads DESIGN, or replace package banners by the line `... some more lines with package banner(s) ...`.